### DISCUSSION TITLE
Azure TRE Accelerator Community Meetup - 09 Oct

### DISCUSSION THREAD

**Discussion by CCCowley** [microsoft/AzureTRE#4102] [General]
We're looking forward to the next Azure TRE Accelerator Community Meetup on 9th October at 4pm BST, and this time we're putting the spotlight on Airlock.

Given all the recent activity around Airlock in the accelerator - both in terms of development work and issues that have been raised - it would be great to gather the community's insights on how to take this element of the Azure TRE forwards.

Register here: https://events.teams.microsoft.com/event/4693d3f0-fb2e-4a78-a55e-44fafe06acad@1fb496e0-1eea-4fbd-be38-d4b9cda1d791

**Agenda highlights:**

1. Airlock discussion - challenges, improvements, and optimisations
2. Latest development updates
3. Roadmap plans and community input

If you've been working on Airlock improvements or have encountered any challenges, we'd love to hear from you. There's no need for formal presentations, just come along and join in the discussion.

If you have a demo you'd like to share, please let us know so we can allocate time.

The Azure TRE Community is central to driving the project forward. We hope you can join on the 9th October!

**Comment by allabright** [comment]
Hi @CCCowley, just wondering if there is a Youtube video for this meetup?

**Reply by CCCowley** [reply]
Hi @allabright, 

Here's the recording from this Meetup: 
https://youtu.be/x-p_usL5_cc?si=EZY8qQqXZV0mXAk0

Thanks!
